lib/publication_delay_report.rb

```diff
@@ -0,0 +1,42 @@
+class PublicationDelayReport
+  def call
+    write_csv($stdout)
```

Comment:
I think it's probably better in this case to pass a target into `#call`, and have the Rake task pass `$stdout` in. Tiny nitpick ;)

Reply:
Sure! I've amended and pushed that